.travis.yml

```diff
@@ -5,13 +5,27 @@ language: node_js
 node_js:
   - 'stable'
 
+env:
+  # Keep star at top so it runs first - it'll be the longest task
+  - PROJECT="*"
```

Comment:
It is only covering the case someone fails to add their project to travis.yml which I can very much see happening. But beyond there is very little use to this granted. Shall I remove it?